src/components/component/CarouselSilder.jsx

```diff
@@ -0,0 +1,74 @@
+import Carousel from "react-material-ui-carousel";
+import mainSlider1 from "../../images/mainSlider1.png";
+import mainSlider2 from "../../images/mainSlider2.png";
+import mainSlider3 from "../../images/mainSlider3.png";
+import { Box, styled } from "@mui/material";
+
+const CarouselSilder = () => {
+  const McyImges = [
```

Comment:
```suggestion
  const McyImgs = [
```
McyImgs로 하던지 McyImages로 하던지 지금은 오타인듯